state/carmen_test_enviroment.go

```diff
@@ -90,13 +90,13 @@ func MakeRandomByteSlice(t *testing.T, bufferLength int) []byte {
 	return buffer
 }
 
-func GetRandom(rangeLower int, rangeUpper int) int {
+func GetRandom(rangeLower int, rangeUpper int) uint64 {
```

Comment:
Should the lower and upper limit of the range also be updated to `uint64` to match the result?

Odd results may be produced from calls like `GetRandom(-10,10)`.

Reply:
With all possible type conversions, I decided to avoid them and throw errors if invalid numbers are passed.